### ISSUE TITLE
addCustomHeader("key","value") is not working when send reinvite for video call in Android. 

### ISSUE THREAD

**Issue by dharmeshpatel1995** [BelledonneCommunications/linphone-android#1776]
I am trying to add a custom header in the video call(Android app, linphone sdk 4.2) invite but at receiver side header is not working.

//File name CallManager
```
public boolean reinviteWithVideo(Context ctx) {
        Core lc = LinphoneManager.getLc();
        Call lCall = lc.getCurrentCall();
        if (lCall == null) {
            Log.e("Trying to reinviteWithVideo while not in call: doing nothing");
            return false;
        }
       // I am trying to add this header but this is not working. 
        CallParams params = lc.createCallParams(lCall);
        params.addCustomHeader("ServerIp",new PrefManager(ctx).getSipDomain());

        if (params.videoEnabled()) return false;
        getBandwidthManager().updateWithProfileSettings(params);

        // Abort if not enough bandwidth...
        if (!params.videoEnabled()) {
            return false;
        }
        lc.updateCall(lCall, params);
        return true;
    }
```

**Comment by Viish** [comment]
Hi,

Please attach logs of the call (from both sides) showing the issue.
Also note that 4.2 SDK is very very old, you should update to 5.1 first, it may solve your issue if the bug was in our lib.

Cheers,

**Comment by dharmeshpatel1995** [comment]
@Viish Thanks for the guidance.

**Comment by Viish** [comment]
Hi @dharmeshpatel1995,
Indeed there is an issue in our SDK for this scenario, thanks for finding it!
Fix is on the way, will be available soon.

**Comment by Viish** [comment]
Fix has been merged in both release/5.1 and release/5.2 SDK branches.
